lib/camera_screen.dart

```diff
@@ -142,10 +137,9 @@ class _CameraScreenState extends State<CameraScreen> {
           return;
         }
 
-        await Navigator.of(context).push(
-          MaterialPageRoute(builder: (context) => DisplayPictureScreen(imagePath: image.path))
-        );
-      } 
+        await Navigator.of(context).push(MaterialPageRoute(
+            builder: (context) => DisplayPictureScreen(imagePath: image.path)));
+      }
```

Comment:
Also merge conflict, should be:
 
```suggestion
        await Navigator.of(context).push(MaterialPageRoute(
            builder: (context) => ImageEditScreen(imagePath: image.path)));
      }
```